raster/src/main/scala/geotrellis/raster/io/geotiff/writer/TiffTagFieldValue.scala

```diff
@@ -77,6 +79,30 @@ object TiffTagFieldValue {
     fieldValues += TiffTagFieldValue(PlanarConfigurationTag, ShortsFieldType, 1, PlanarConfigurations.PixelInterleave)
     fieldValues += TiffTagFieldValue(SampleFormatTag, ShortsFieldType, 1, imageData.bandType.sampleFormat)
 
+    if(geoTiff.options.colorSpace == ColorSpace.Palette) {
+
+      val bitsPerSample = imageData.bandType.bitsPerSample
```

Comment:
Unclear whether to throw here if it's invalid to have for this BandType, or on creation. I would say perhaps on creation.